static/index.html

```diff
@@ -104,6 +104,18 @@
 				</div>
 			</div>
 		</div>
+		<div class="mb-2">
+			<div class="row">
+				<div class="col-8">
+					<div class="input-group mb-3">
+						<input type="number" id="refreshRate" class="form-control" value="10">
+						<div class="input-group-append">
+							<span class="input-group-text">Seconds</span>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
```

Comment:
Perhaps I'm a bit picky, but I feel like a small drop-down aligned to right right would be cuter.

Reply:
Suggestions have been added